app/views/users/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for @user do |u| %>
+  <% if @user.errors.any? %>
```

Comment:
The errors canactually go before the form as they're usually displayed on top of the form. Furthermore, you can actually store this form inside a partial `_form.html.erb` and just render it in `new` and `edit`.

[`OPTIONAL`]